app/aem/core/src/main/kotlin/com/cognifide/apm/core/grammar/ScriptRunner.kt

```diff
@@ -35,20 +35,23 @@ import com.cognifide.apm.core.grammar.utils.ImportScript
 import com.cognifide.apm.core.grammar.utils.RequiredVariablesChecker
 import com.cognifide.apm.core.logger.Position
 import com.cognifide.apm.core.logger.Progress
+import com.cognifide.apm.core.method.MethodInvoker
```

Comment:
mixing kt with java is a mess, are you planning to remove kt ?

Reply:
yes, I'm planning that